microcosm_postgres/tests/test_serial_type.py

```diff
@@ -56,11 +56,12 @@ def test_retrieve_sequence_value(self):
             with transaction():
                 example = self.store.create(WithSerial())
 
+            value = example.value
             context.session.expunge(example)
 
             example = self.store.retrieve(example.id)
 
-        assert_that(example.value, is_(equal_to(1)))
+        assert_that(example.value, is_(equal_to(value)))
```

Comment:
Since we're no longer resetting sequences, the unit tests here now just test relative values.